docs/source/onedal/algorithms/statistics/basic-statistics.rst

```diff
@@ -65,6 +65,68 @@ the problem is to compute the following sample characteristics for each feature
    * - Variation coefficient
      - :math:`V(j) = \frac {\text{stdev}(j)} {m(j)}`
 
+.. _basic_statistics_p_math:
+
+Partial Computing
+-----------------
+
+Given a block of a dataset :math:`X = \{ x_1, \ldots, x_n \}` with :math:`n` feature vectors of dimension :math:`p`,
+the sums is a :math:`1 \times p` matrix, the crossproduct is :math:`p \times p` square matrices.
```

Comment:
```suggestion
the sum is a :math:`1 \times p` matrix, the cross product is :math:`p \times p` square matrices.
```

Reply:
Does it make sense to use sums, due to here, it means a vector of sum values?

Reply:
yes, in that case, it's fine